graphene_django/tests/test_query.py

```diff
@@ -1586,3 +1587,81 @@ class Query(graphene.ObjectType):
         "allReporters": {"edges": [{"node": {"firstName": "Jane", "lastName": "Roe"}},]}
     }
     assert result.data == expected
+
+
+def test_should_query_django_objecttype_fields_custom_meta():
+    class ArticleTypeOptions(DjangoObjectTypeOptions):
+        """Article Type Options with extra fields"""
+
+        fields = yank_fields_from_attrs(
+            {"headline_with_lang": graphene.String()}, _as=graphene.Field,
+        )
+
+    class ArticleBaseType(DjangoObjectType):
+        class Meta:
+            abstract = True
+
+        @classmethod
+        def __init_subclass_with_meta__(cls, **options):
+            options.setdefault("_meta", ArticleTypeOptions(cls))
```

Comment:
I wonder if this is the intended way to override `_meta` and why it can't be part of `ArticleBaseType.Meta`.

Reply:
Actually after implementing my option for Parler. Probably I can modify this test. Basically in the `__init_subclass_with_meta__` we can inject more fields that are part of the model but are not part of the Meta fields of the Django model.

You can see the real life example I put in the comments. I can modify this example to match that one.

Reply:
@zbyte64 I updated the test with a more realistic scenario. Please let me know what do you think :)